python/paddle/fluid/incubate/fleet/collective/__init__.py

```diff
@@ -271,7 +271,6 @@ def _try_to_compile(self, startup_program, main_program):
         node_num = self._node_num()
         assert node_num >= 1, "nccl2 node_num must >= 1, now:{}" % node_num
 
-        self._strategy.fuse_all_reduce_ops = True
```

Comment:
This line of code forces fuse_all_reduce_ops = True, regardless of whether the user has previously set it or not.